llama_stack/providers/inline/safety/prompt_guard/prompt_guard.py

```diff
@@ -75,7 +75,7 @@ def __init__(
         self.temperature = temperature
         self.threshold = threshold
 
-        self.device = "cuda"
+        self.device = self.config.guard_execution_type
```

Comment:
can we just check if cuda is available and use that otherwise use CPU? no need for a specific configuration like this to be added.